### PR TITLE
DROOLS-6703 DMN TCK PRs instructions

### PR DESCRIPTION
**Pull request by tarilabs**
**JIRA**: https://issues.redhat.com/browse/DROOLS-6703

**referenced Pull Requests**: none

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>

* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>

* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: kie-dmn/kie-dmn-tck/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+In the remainder of this documentation, the following conventions are assumed:
+
+```
+TCK repository URL: https://github.com/dmn-tck/tck
+default TCK branch: master
+```
+
+## Testing stable DMN TCK tests locally
+
+You can test this module locally by making use of the appropriate maven profile `-Ddmn-tck`.
```
Comment (tkobayas) [inline kie-dmn/kie-dmn-tck/README.md:24]: This should be `-Pdmn-tck` :)

Comment (tkobayas) [inline kie-dmn/kie-dmn-tck/README.md]: This should be `-P` :)
```suggestion
You can test this module locally by making use of the appropriate maven profile `-Pdmn-tck`.
```

**Review by tkobayas — APPROVED** [review]
Thank you, Matteo. This is very useful for contributors!

File: kie-dmn/kie-dmn-tck/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# kie-dmn-tck
+
+This is a Maven utility module for DMN TCK support; this module is used to integrate **stable DMN TCK tests** inside of kie-dmn CI.
```
Comment (danielezonca) [inline kie-dmn/kie-dmn-tck/README.md:3]: What about add the meaning of the acronym TCK?

File: kie-dmn/kie-dmn-tck/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+## Testing stable DMN TCK tests locally
+
+You can test this module locally by making use of the appropriate maven profile `-Ddmn-tck`.
+
+NOTE: no TCK tests will be run without enabling the appropriate Maven profile.
+
+Please reference the configuration of `pom.xml` for more details.
+
+## Testing new DMN TCK tests contributions
```
Comment (danielezonca) [inline kie-dmn/kie-dmn-tck/README.md]: I have found a bit misleading this last chapter: it is the readme of `kie-dmn-tck` module but it describes a procedure that is manual and "unrelated" with it.
What about making this more explicit with something like
```suggestion
## Testing new DMN TCK tests contributions (without `kie-dmn-tck`)
```
?
As alternative I don't know if it is possible to describe how to use this module also for PR testing: probably it is enough to override some [properties](https://github.com/kiegroup/drools/blob/main/kie-dmn/kie-dmn-tck/pom.xml#L35) using the source repo of the PR but I don't know how to replace git rebase using `maven-scm-plugin` 🤔 

Feel free to ignore this second part of the comment, my main point is the first part about making clear that this is "covered" by `kie-dmn-tck`

File: kie-dmn/kie-dmn-tck/README.md (hunk 1 above)
Comment (tarilabs) [reply kie-dmn/kie-dmn-tck/README.md]: well spotted! thanks

File: kie-dmn/kie-dmn-tck/README.md (hunk 2 above)
Comment (tarilabs) [reply kie-dmn/kie-dmn-tck/README.md:3]: this is referenced here: 

https://github.com/kiegroup/drools/pull/3995/files#diff-6d3a597a9e8d40a6e66271f2f6e2916770df38b9f8d2c771c944d9bb60cad9c8R9

**Review by danielezonca — APPROVED** [review]

File: kie-dmn/kie-dmn-tck/README.md (hunk 3 above)
Comment (tarilabs) [reply kie-dmn/kie-dmn-tck/README.md]: added 643c861

**Pull request merged**